frontend/src/layout/navigation/TopBar/Announcement.tsx

```diff
@@ -41,7 +40,7 @@ export function Announcement(): JSX.Element | null {
             </div>
         )
     } else if (shownAnnouncementType === AnnouncementType.CloudFlag && cloudAnnouncement) {
-        message = <ReactMarkdown className="strong">{cloudAnnouncement}</ReactMarkdown>
```

Comment:
I mean, `dangerouslySetInnerHTML` _should_ be safe here considering this comes from our production project, but why not keep Markdown for the UX? The main issue previously was that before payloads those feature flag variants were limited in format and Markdown wasn't super feasible, but now this should be really smooth.

Reply:
I'm ambivalent. The reason was to allow more expressive things, since payloads are a lot more flexible than just markdown now.

At the same time, I don't see much more usage than a link, emoji and text

Reply:
changed it back!